src/models/monster.ts

```diff
@@ -0,0 +1,12 @@
+export type Monster = {
+    name: string
+    xp: number
+    environments: Environment[]
+}
+
+export type Environment = 'MOUNTAIN' | 'TREE'
+
+export const Monsters: Monster[] = [
+    { name: 'Goblin', environments: ['MOUNTAIN'], xp: 25 },
+    { name: 'Frost Giant', environments: ['MOUNTAIN'], xp: 2900 },
+]
```

Comment:
I wonder how we get the full set here. There's the [SRD ](https://dnd.wizards.com/articles/features/systems-reference-document-srd) which I think is free to use but I don't know if its accessible as an API.